chips/cc26xx/src/uart.rs

```diff
@@ -0,0 +1,218 @@
+//! UART driver, cc26xx family
+use core::cell::Cell;
+use gpio;
+use ioc;
+use kernel;
+use kernel::common::regs::{ReadOnly, ReadWrite, WriteOnly};
+use kernel::hil::gpio::Pin;
+use kernel::hil::uart;
+use prcm;
+
+pub const UART_BASE: usize = 0x4000_1000;
+pub const MCU_CLOCK: u32 = 48_000_000;
+
+#[repr(C)]
+struct Registers {
+    dr: ReadWrite<u32>,
+    rsr_ecr: ReadWrite<u32>,
+    _reserved0: [u8; 0x10],
```

Comment:
It's a little weird to me that these are `u8` rather than `[u32; 0x4]`? If it's a matching the datasheet thing you can tell me to go away :)